### DISCUSSION TITLE
vr-sros change CLI config mode

### DISCUSSION THREAD

**Discussion by mohamedhafez87** [srl-labs/containerlab#1408] [Q&A]
Hi,

Is there an option available to include the CLI config mode in the YAML file, which would allow for changing the CLI mode before deploying the lab?

Thank you.

**Comment by hellt** [comment]
Hi @mohamedhafez87 
We default to Model-Driven mode (MD-CLI) for all version that support it.
If you still require classic, then you have to manually switch the engine after the lab is deployed

**Reply by mohamedhafez87** [reply]
But how can I use the "--mode" option that is already mentioned in vrnetlab/launch.py? Can it be added as an environment variable in the node context?

**Reply by hellt** [reply]
This flag is not implemented still
